### DISCUSSION TITLE
control griper in motion_gen_example.py

### DISCUSSION THREAD

**Discussion by peter05010402** [NVlabs/curobo#47] [Software Q&A] [answered]
Hi,
How to control the griper open/closed in motion_gen_example.py ?

**Comment by balakumar-s** [comment, marked as answer]
We do not have examples of moving the gripper in the motion_gen_example.py. You could try to move the gripper joints manually. Alternatively, you can look at this example: https://github.com/NVlabs/curobo/blob/4aabfa020601f9c50e34df6517e0916faf0dfd32/examples/isaac_sim/simple_stacking.py#L504 which uses a Franka object to command grippers. 

Some notes on the block stacking example: https://curobo.org/source/advanced_examples/2b_block_stacking_example.html

**Comment by peter05010402** [comment]
Thank you for your reply.  
It's a good idea to move the gripper joints.
